src/main/kotlin/uk/gov/dluhc/printapi/database/entity/Certificate.kt

```diff
@@ -86,6 +87,12 @@ class Certificate(
      */
     var initialRetentionDataRemoved: Boolean = false,
 
+    /**
+     * The date that all remaining certificate data should be removed after the third (final) retention period. This is
+     * currently specified as the tenth 1st July in the legislation.
```

Comment:
`as the tenth 1st July ` ? (Similar comment in file above). Is that a UK legislation called `tenth 1st July`?

Reply:
It's ten (10 x) 1st Julys from now 🤯